.github/workflows/publish_build.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Binaries
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2 
+        with:
+          persist-credentials: false
+
+      - name: Install and Build 🔧
+        run: |
+          npm install
+          npm run build
+          tar -czf ipfs-webui.tar.gz build/*
+
+      - name: Copy build-artifacts to Release
+        uses: skx/github-action-publish-binaries@master
```

Comment:
Please pin `skx/github-action-publish-binaries` to a specific version

Reply:
Sorry missed this comment pinned to @release-0.14